crypto/ec/ecp_nistp521.c

```diff
@@ -43,6 +43,11 @@
 #include <openssl/err.h>
 #include "ec_local.h"
 
+#define STRICT_ALIGNMENT 1
```

Comment:
Hmm, this is x86 only, right?
but STRICT_ALIGNMENT is ARMv4 and PowerPC.

Reply:
So I would kindly ask for a different name of this define.

Reply:
This is how it is done elsewhere, whitelisting most platforms but not all of them:
https://github.com/openssl/openssl/blob/07caec83b81859ea9aa2d5075a394aa48c4e5fae/include/crypto/modes.h#L30-L39

Would this be acceptable to you?

Reply:
Well, I thought about that, but this moule module is only x86 ?